apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -1,178 +1,179 @@
 defmodule AndiWeb.EditLiveView do
-  use Phoenix.LiveView
-  alias Phoenix.HTML.Form
+  use AndiWeb, :live_view
+
   alias Phoenix.HTML.Link
```

Comment:
If you do keep `Phoenix.HTML.Link` in the live_view macro you won't need this here.